sdk/media/azure-media-lva-edge/README.md

```diff
@@ -0,0 +1,127 @@
+# Azure Live Video Analytics for IoT Edge client library for Python
+
+Live Video Analytics on IoT Edge provides a platform to build intelligent video applications that span the edge and the cloud. The platform offers the capability to capture, record, and analyze live video along with publishing the results, video and video analytics, to Azure services in the cloud or the edge. It is designed to be an extensible platform, enabling you to connect different video analysis edge modules (such as Cognitive services containers, custom edge modules built by you with open-source machine learning models or custom models trained with your own data) to it and use them to analyze live video without worrying about the complexity of building and running a live video pipeline.
+
+Use the client library for Live Video Analytics on IoT Edge to:
+
+- Simplify interactions with the [Microsoft Azure IoT SDKs](https://github.com/azure/azure-iot-sdks) 
+- Programatically construct media graph topologies and instances
+
+[Package (PyPi)][package] | [Product documentation][doc_product] | [Direct methods][doc_direct_methods] | [Media graphs][doc_media_graph] | [Source code][source] | [Samples][samples]
+
+## Getting started
+
+### Install the package
+
+Install the Live Video Analytics client library for Python with pip:
+
+```bash
+pip install azure-lva-edge
+```
+### Prerequisites
+
+* Python 2.7, or 3.5 or later is required to use this package.
+* You need an [Azure subscription][azure_sub], and a [IOT device connection string][iot_device_connection_string] to use this package.
```

Comment:
active Azure subscription.

Also, does the user need to have owner privileges or can they work with a Contributor role?